addons/web/static/src/js/views/form/form_renderer.js

```diff
@@ -1067,15 +1067,18 @@ var FormRenderer = BasicRenderer.extend({
         var self = this;
 
         // Set the new content of the form view, and toggle classnames
-        this.$el.html($newContent);
+        this.el.innerHTML = "";
         this.$el.toggleClass('o_form_nosheet', !this.has_sheet);
-        if (this.has_sheet) {
-            this.$el.children().not('.o_FormRenderer_chatterContainer')
-                .wrapAll($('<div/>', {class: 'o_form_sheet_bg'}));
-        }
         this.$el.toggleClass('o_form_editable', this.mode === 'edit');
         this.$el.toggleClass('o_form_readonly', this.mode === 'readonly');
 
+        const $content = this.has_sheet ?
+            $(this._wrapFormContents(...$newContent)) :
+            $newContent;
+        dom.append(this.$el, $content, {
+            in_DOM: this._isInDom,
+            callbacks: [{ widget: this }],
```

Comment:
this is incorrect: if `this._isInDom` is true, `on_attach_callback` will be called on the renderer, even though it's already in the DOM. Could you confirm?